### PR TITLE
Update german translations

### PR DESCRIPTION
**Pull request by System98**
**When merged this pull request will:**

1. Fix some german translations like discussed in PR #3423 
2. Re-add strings in "medical" addon deleted in PR #3437 
3. Add german translations for those previously deleted strings and update the english texts so the purpose of the function gets clearer

Please review and check if the

STR_ACE_Medical_delayUnconCaptive
STR_ACE_Medical_MedicalSettings_delayUnconCaptive_Description
STR_ACE_Medical_MedicalSettings_delayUnconCaptive_DisplayName

string texts (english & german) in "medical" are easier to understand now and help understand the purpose of the function.

### CODE REVIEW

File: addons/medical/stringtable.xml [hunk 1]
```diff
@@ -4108,14 +4108,22 @@
...
+            <German>Verzögert Ende des KI-Beschusses bei medizinischer Bewustlosigkeit</German>
             <Spanish>Demora antes de volverse neutral al caer inconsciente</Spanish>
             <Polish>Opóźnij status captive u nieprzytomnych osób</Polish>
         </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_delayUnconCaptive_Description">
+            <English>Delay cease fire of AI while player is unconscious for medical reasons.</English>
+            <German>Verzögert das Ende des KI-Beschusses auf einen Spieler, wenn dieser aus medizinischen Gründen bewustlos wird.</German>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_delayUnconCaptive_DisplayName">
+            <English>Delay cease fire of AI during medical unconsciousness</English>
```
Comment (Glowbal) [inline addons/medical/stringtable.xml]: This is to long for a settings display name

Comment (Glowbal) [inline addons/medical/stringtable.xml]: Replace `during medical` with `for` perhaps?

Comment (System98) [inline addons/medical/stringtable.xml]: Changed display name according to your suggestion. Is the description length ok or does it need shortening too?

Comment (SilentSpike) [inline addons/medical/stringtable.xml]: Didn't we remove this setting? :confused:

File: addons/medical/stringtable.xml [hunk 2]
```diff
@@ -4108,14 +4108,22 @@
...
         </Key>
         <Key ID="STR_ACE_Medical_delayUnconCaptive">
-            <English>Delay Medical Unconscious Captivity</English>
-            <German>Verzögert Gefangenschaft bei medizinischer Bewustlosigkeit</German>
+            <English>Delay cease fire of AI during medical unconsciousness</English>
+            <German>Verzögert Ende des KI-Beschusses bei medizinischer Bewustlosigkeit</German>
             <Spanish>Demora antes de volverse neutral al caer inconsciente</Spanish>
             <Polish>Opóźnij status captive u nieprzytomnych osób</Polish>
         </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_delayUnconCaptive_Description">
```
Comment (jonpas) [inline addons/medical/stringtable.xml]: Need to be added to CfgVehicles.hpp and ACE_Settings.hpp, currently not used.

File: addons/medical/stringtable.xml [hunk 3]
```diff
@@ -4108,14 +4108,22 @@
...
+            <English>Delay cease fire of AI while player is unconscious for medical reasons.</English>
+            <German>Verzögert das Ende des KI-Beschusses auf einen Spieler, wenn dieser aus medizinischen Gründen bewustlos wird.</German>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_delayUnconCaptive_DisplayName">
```
Comment (jonpas) [inline addons/medical/stringtable.xml]: Need to be added to CfgVehicles.hpp and ACE_Settings.hpp, currently not used.

**Comment by Glowbal** [comment]
We will sort out the CfgVehicle entries in a different PR.

**Pull request merged**